### PR TITLE
Fix clustering model entrance

### LINKED ISSUE #870: Metric 'accuracy' should be remove when compiling deep_embedding_cluster
**Description**
The metric of `accuracy` should be removed when compiling cluster model since there is no `accuracy` to evaluate performance of unsupervised clustering when training. 

**Screenshots**
image

image

### LINKED ISSUE #871: Don't output training info when using cluster model.
**Description**
The cluster model didn't output any info during train process. However, 'print' in `template_tf.go` did work.

### PR DESCRIPTION
**Pull request by weiguoz**
Fix #871 
Fix #870

follow [deep_embedding_cluster.py::sqlflow_train_loop](https://github.com/sql-machine-learning/models/blob/73adf6a0def373ef477e4877aa81d107ee6d86f5/sqlflow_models/deep_embedding_cluster.py#L191-L196)

### CODE REVIEW

File: sql/template_tf.go [hunk 1]
```diff
@@ -127,15 +127,18 @@ def validate_input_fn(batch_size):
 classifier.compile(optimizer=classifier.default_optimizer(),
     loss=classifier.default_loss(),
     metrics=["accuracy"])
-if hasattr(classifier, 'cluster_train_loop'):
-    classifier.cluster_train_loop(train_input_fn(BATCHSIZE))
+premade = hasattr(classifier, 'sqlflow_train_loop')
```
Comment (Yancey0623) [inline sql/template_tf.go]: `sqlflow_train_loop ` is used to run the custom train loop instead of **premade**

File: sql/template_tf.go [hunk 2]
```diff
@@ -127,15 +127,18 @@ def validate_input_fn(batch_size):
...
+    classifier.sqlflow_train_loop(train_input_fn(BATCHSIZE))
 else:
     classifier.fit(train_input_fn(BATCHSIZE),
         epochs=EPOCHS if EPOCHS else classifier.default_training_epochs(),
         verbose=VERBOSE)
 classifier.save_weights("{{.Save}}", save_format="h5")
 eval_result = classifier.evaluate(validate_input_fn(BATCHSIZE), verbose=VERBOSE)
-print("Training set accuracy: {accuracy:0.5f}".format(**{"accuracy": eval_result[1]}))
+
+if not premade:
```
Comment (Yancey0623) [inline sql/template_tf.go]: I think this condition can be the `label clause` is empty, no need to run the evaluation if there is no `label clause`:

``` golang
if "{{.Y.FeatureName}}" != "":
    eval_result = classifier.evaluate(validate_input_fn(BATCHSIZE), verbose=VERBOSE)
    print("Training set accuracy: {accuracy:0.5f}".format(**{"accuracy": eval_result[1]}))
```

Comment (weiguoz) [reply sql/template_tf.go]: Good idea.

Comment (weiguoz) [reply sql/template_tf.go]: It looks SQLFlow does not need to split the dataset into [train&val](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/design/design_training_and_validation.md) while `Y.FeatureName` is empty.

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**